src/Build/GlobalAssemblyInfo.cs

```diff
@@ -1,3 +1,29 @@
+// Project Orleans Cloud Service SDK ver. 1.0
```

Comment:
I'm a bit confused - what's the point of reformatting the license header in every source file?